service/apiserver/internal/logic/account/getaccountlogic.go

```diff
@@ -76,6 +76,7 @@ func (l *GetAccountLogic) GetAccount(req *types.ReqGetAccount) (resp *types.Acco
 		Pk:     account.PublicKey,
 		Nonce:  account.Nonce,
 		Assets: make([]*types.AccountAsset, 0),
+		Lps:    make([]*types.AccountLp, 0),
```

Comment:
init with `cap` will be more effective when doing append. `Lps: make([]*types.AccountLp, 0, ln(account.AsstInfo)`

Reply:
good suggestion. updated this one and other ones.